hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -122,7 +121,9 @@ public int getVersion() {
 
 	@Override
 	public int getMinimumChildCount(int version) {
-		return ChildIndices.NUM_V1_CHILDREN;
+		return (version == RELEASE_070_VERSION)
+				? ChildIndices.NUM_070_CHILDREN
+				: ChildIndices.NUM_080_CHILDREN;
```

Comment:
I suppose there will be a better mechanism to handle the different version issue in the future if yet another new version needs to be added.

Reply:
True...though most likely we will never need to worry about more than two versions at a time (the previous release and the current release).

Reply:
agreed.